rfc6962-bis.xml

```diff
@@ -610,8 +628,7 @@ protocol to which the SCT conforms. This version is v2. Note that <xref target="
           <spanx style="verb">signed_entry</spanx> includes the TBSCertificate from either the <spanx style="verb">leaf_certificate</spanx> (in the case of an X509ChainEntry) or the <spanx style="verb">pre_certificate</spanx> (in the case of a PrecertChainEntryV2).
         </t>
         <t>
-          <spanx style="verb">extensions</spanx> are future extensions to
-SignedCertificateTimestamp v2. Currently, no extensions are specified.
+          <spanx style="verb">extensions</spanx> are future extensions to SignedCertificateTimestamp v2. Currently, no extensions are specified. If an implementation sees an extension that it does not understand, it SHOULD ignore that extension. Furthermore, an implementation MAY choose to ignore any extension(s) that it does understand.
```

Comment:
Should we also explain that if there's ever a need to define an SCT "extension" that clients MUST understand and process, the way to accommodate this will be to bump the SCT version and change the SCT structure?